rnd/market/market/routes/admin.py

```diff
@@ -32,10 +40,16 @@ async def create_agent_entry(request: market.model.AddAgentRequest):
 
 
 @router.post("/agent/featured/{agent_id}")
-async def set_agent_featured(agent_id: str, category: str = "featured"):
+async def set_agent_featured(
+    agent_id: str,
+    category: str = "featured",
+    user: prisma.models.User = fastapi.Depends(market.auth.get_user),
+):
     """
     A basic endpoint to set an agent as featured in the database.
     """
```

Comment:
I think there’s a way to set a decorator for this. Seems easier to scale for 

@required_role(“admin”)

Reply:
@requires_loggedin as well for some maybe

Reply:
Okay thank you, I'll look into it